### PR TITLE
Remove the deprecated package optparse to use argparse instead. close #9

### LINKED ISSUE #9: optparse is deprecated
The [optparse](https://docs.python.org/3/library/optparse.html) module is deprecated since version version 3.2 (in Python 3.x) and 2.7 (in Python 2.x). Instead we should use the relative module [argparse](https://docs.python.org/3/library/argparse.html#module-argparse).

I want to use the argparse in another project that I'm developing for a meetup. So after that I could do that in Pygalume too.

### PR DESCRIPTION
**Pull request by mazulo**
Ready for review and merge

### CODE REVIEW

File: pygalume/controller/database.py [hunk 1]
```diff
@@ -2,15 +2,15 @@
 
 from sqlalchemy import and_
 
-from myexceptions import MusicNotFound, ArtistNotFound
+# from myexceptions import MusicNotFound, ArtistNotFound
```
Comment (delete) [inline pygalume/controller/database.py]: If is not using, it is better remove, let the code cleaner.

Comment (mazulo) [inline pygalume/controller/database.py]: Indeed

**Comment by delete** [comment]
Awesome work bro! 
Can we merge into master and let the world now about pygalume now? :D

**Comment by mazulo** [comment]
OH YEAH!

**Pull request merged**